### ISSUE TITLE
README mismatch to UpgradeAlert API

### ISSUE THREAD

**Issue by azack** [larryaasen/upgrader#140]
Minor but the README.md at 
https://github.com/larryaasen/upgrader/blob/0c4cb966413dca802321ae5b3add3786647bf2cf/README.md#L105

describes the `UpgradeAlert` as taking a `debugDisplayAlways` parameter, but the parameter name is actually `debugAlwaysUpgrade`: https://github.com/larryaasen/upgrader/blob/0c4cb966413dca802321ae5b3add3786647bf2cf/lib/src/upgrade_alert.dart#L20

```
  upgrader:
    dependency: "direct main"
    description:
      name: upgrader
      url: "https://pub.dartlang.org"
    source: hosted
    version: "3.8.0"
```

**Comment by larryaasen** [comment]
Thanks for pointing that out. I will get this resolved.

**Comment by larryaasen** [comment]
@azack This was fixed in this commit: f387c490e4109f4a7a660b006af4214f55b4a914, and will be included in the next release, 3.14.0.
